config/boxes/venues.json

```diff
@@ -0,0 +1 @@
+{"name":"Private Events Venues","feature":"private_event_venues","engine":"venue","has_featured":true,"has_locations":true}
```

Comment:
I stoled this config from sensei, I tested locally seem to work but not sure if all properties make sense in this context 🤔 .